scripts/eval/rag_evaluator/ragas_metrics.py

```
@@ -0,0 +1,395 @@
+                )
+                req = self.llm.agenerate_text(
+                    formatted_prompt,
+                    n=1,
+                    temperature=0.1,
+                )
+                resp = await req
+                score0 = self.process_score(resp.generations[0][0].text)
+                if score0 == score0:
+                    break
+
+            for retry in range(5):
```

Ruff (0.15.10)

[warning] Loop control variable `retry` not used within loop body

Rename unused `retry` to `_retry`

(B007)